studiopress/pattern-manager
# [Don't merge] Lazy loading with `<iframe loading="lazy">`

Just to test `<iframe loading="lazy">`. 

### How to test

1. Activate Frost
2. Load the Pattern Manager app
3. Observe the lazy loading behavior
